RULES_DESCRIPTIONS.md

```diff
@@ -233,12 +233,14 @@ _Configuration_: N/A
 ## defer
 
 _Description_: This rule warns on some common mistakes when using `defer` statement. It currently alerts on the following situations:
+
```

Comment:
I think this will fix the current rendering issue:
<img width="1200" alt="Screen Shot 2022-07-23 at 8 33 11 PM" src="https://user-images.githubusercontent.com/77197/180628767-b42b4901-4630-4d2d-a3a2-c8e7b9343d6c.png">